jenkins/repo_health_reports.py

```diff
@@ -0,0 +1,119 @@
+"""
+Script to help create a PR with Repo health report. To be run inside
+a Jenkins job that first runs the health checks on a target repo
+(see pytest-repo-health and edx-repo-health repos)
+"""
+from __future__ import absolute_import
+
+import logging
+
+import click
+import six
+from github import GithubObject
+
+from .github_helpers import (authenticate_with_github, branch_exists,
+                             close_existing_pull_requests, connect_to_repo,
+                             create_branch, create_pull_request,
+                             get_modified_files_list, update_list_of_files)
+
+logging.basicConfig()
+LOGGER = logging.getLogger()
+LOGGER.setLevel(logging.INFO)
+
+
+@click.command()
+@click.option(
+    '--sha',
+    help="Sha of the merge commit to base the new PR off of",
+    required=True,
+)
+@click.option(
+    '--repo_root',
+    help="Path to local repository to run repo health on. "
+    "Make sure the path includes the repo name",
+    required=True,
+)
+@click.option(
+    '--org',
+    help="The github organization for the repository to run make upgrade on.",
+    required=True,
+)
+@click.option(
+    '--user_reviewers',
+    help="Comma seperated list of Github users to be tagged on pull requests",
+    default=None
+)
+@click.option(
+    '--team_reviewers',
+    help="Comma seperated list of Github teams to be tagged on pull requests",
+    default=None
+)
+def main(sha, repo_root, org, user_reviewers, team_reviewers):
+    """
+    Inspect the results of running ``make upgrade`` and create a PR with the
+    changes if appropriate.
+    """
+    LOGGER.info("Authenticating with Github")
+    github_instance = authenticate_with_github()
+
+    # Last folder in repo_root should be the repository
+    directory_list = repo_root.split("/")
+    repository_name = directory_list[-1]
+    LOGGER.info(u"Trying to connect to repo: {}".format(repository_name))
```

Comment:
Is there a reason to prefix this with the Unicode literal?

Reply:
nope, copy-pasta leftovers